### PR TITLE
Add "liens utiles" to student services

### PR DESCRIPTION
**Pull request by Shinyhero36**
## Issue
#32 

## Screenshot
Screenshot_1658997493

## Notes
- J'ai repris le même icône que celui pour WikiEtud (à voir s'il faut mettre autre chose si c'est redondant)

### CODE REVIEW

File: src/utils/Services.ts [hunk 1]
```diff
@@ -196,6 +197,17 @@ export function getStudentServices(
       badgeFunction: (dashboard: FullDashboardType): number =>
         dashboard.available_tutorials,
     },
+    {
+      key: SERVICES_KEY.USEFUL_LINKS,
+      title: 'Liens utiles',
```
Comment (Paulao17) [inline src/utils/Services.ts:202]: Title should use `i18n`

File: src/utils/Services.ts [hunk 2]
```diff
@@ -196,6 +197,17 @@ export function getStudentServices(
...
+      subtitle: i18n.t('screens.services.descriptions.usefulLinks'),
+      image: Urls.images.wiketud,
```
Comment (Paulao17) [inline src/utils/Services.ts:204]: I propose we copy the `Wiketud.png` image on `etud` to something like `LiensUtiles.png` (see [urls](/ClubInfoInsaT/application-amicale/blob/master/src/constants/Urls.tsx) ). That way we can later easily change the image without updating the app.

**Review by Paulao17 — CHANGES_REQUESTED** [review]
Good ! Works well, missing a few locales (translations) in places. Should at least have the French and English translation.

As commented, I propose copying the wiketud image for easier later modification.

**Comment by Shinyhero36** [comment]
Theoretically, every items in services should use i18n in the title. That's why I let it in French.

File: src/utils/Services.ts (hunk 2 above)
Comment (Paulao17) [reply src/utils/Services.ts:204]: Could you please use `https://etud.insa-toulouse.fr/~amicale_app/images/UsefulLinks.png` instead please ?

**Comment by Paulao17** [comment]
Resolves #32

**Pull request merged**